### PR TITLE
Feature cart page

### PR DESCRIPTION
**Pull request by Bhakti199**
here I created cart page

Screenshot (1114)

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
looks good to merge

**Review by Avin008 — COMMENTED** [review]
Everything looks great!. well structured code.

File: cart-page.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+          </div>
+        </div>
+      </div>
+      </div>
+
+      
+
+
+
+
```
Comment (tusharkandpal) [inline cart-page.html]: can remove this extra space

**Review by tusharkandpal — APPROVED** [review]
looks good! just one suggestion mentioned below

**Pull request merged**